oci-unit-tests/standalone-checks_test.sh

```diff
@@ -0,0 +1,91 @@
+# shellcheck shell=dash
+
+# shellcheck disable=SC1090
+. "$(dirname "$0")/helper/test_helper.sh"
+. "$(dirname "$0")/helper/common_vars.sh"
+
+# cheat sheet:
+#  assertTrue $?
+#  assertEquals ["explanation"] 1 2
+#  oneTimeSetUp()
+#  oneTimeTearDown()
+#  setUp() - run before each test
+#  tearDown() - run after each test
+
+# Standalone checks that will be performed for each image.
+
+# Check that all supported architectures are present in the manifest
+# list for every image.
+test_all_supported_architectures_are_available()
+{
+    # We only support querying AWS for now.
+    local registries="aws"
+    local namespaces="ubuntu lts"
+    local utilsdir
+    local listed_architectures
+    local ret=0
+
+    utilsdir="$(mktemp -d)/utils"
+    trap 'rm -rf ${utilsdir}' 0 INT QUIT ABRT PIPE TERM
+
+    debug "Checking if all supported architectures are present in the manifest list of each image"
+
+    if ! git clone -q --depth 1 https://git.launchpad.net/~canonical-server/ubuntu-docker-images/+git/utils "${utilsdir}"; then
+	debug "failed to clone 'utils' repository"
+	rm -rf "${utilsdir}"
+	exit 1
+    fi
+
+
+    # Iterate over the available registries...
+    for registry in ${registries}; do
+        # ... and over the available namespaces...
+        for namespace in ${namespaces}; do
+            # ... and over the list of all images for the
+            # register/namespace combination...
+            for image in $("${utilsdir}"/list-all-images.sh \
+                                        --registry "${registry}" \
+                                        --namespace "${namespace}"); do
+                # ... and over the list of all available tags for each
+                # image...
+                for tag in $("${utilsdir}"/list-tags-for-image.sh \
+                                          --registry "${registry}" \
+                                          --namespace "${namespace}" \
+                                          --image "${image}" \
+                                 | grep '\(latest\|beta\|edge\)'); do
+                    debug "Checking manifest list for ${image}:${tag} on ${registry}"
```

Comment:
Because we're not using the registry URL here, I opted to use the same format as I'm using in the other error message:

```
"E: Could not obtain manifest list for ${namespace}/${image}:${tag} on ${registry}"
```